DOCS.md

```diff
@@ -8,7 +8,189 @@ Registry: https://hub.docker.com/r/target/vela-docker
+| `quiet`                 | enables suppressing the build output and print image ID on success                                                  | `false`  | N/A     |
+| `remove`                | enables removing the intermediate containers after a successful build                                               | `false`  | N/A     |
+| `secrets`               | enables setting a secret file to expose to the build - format (id=mysecret,src=/local/secret)                       | `false`  | N/A     |
+| `security_opts`         | enables setting security options                                                                                    | `false`  | N/A     |
+| `shm_sizes`             | enables setting the size of /dev/shm                                                                                | `false`  | N/A     |
+| `squash`                | enables setting squash newly built layers into a single new layer                                                   | `false`  | N/A     |
+| `ssh_components`        | enables setting an ssh agent socket or keys to expose to the build - format (default|<id>[=<socket>|<key>[,<key>]]) | `false`  | N/A     |
+| `stream`                | enables streaming attaches to server to negotiate build context                                                     | `false`  | N/A     |
+| `tags`                  | enables naming and optionally a tagging - format (name:tag)                                                         | `true`   | N/A     |
+| `target`                | enables setting the target build stage to build.                                                                    | `false`  | N/A     |
+| `ulimits`               | enables setting ulimit options                                                                                      | `false`  | N/A     |
+
+### Daemon
+
+The following parameters are used to configure the image:
+
+| Name                  | Description                                             | Required | Default |
+| --------------------- | ------------------------------------------------------- | -------- | ------- |
+| `bip`                 | enables specifying a network bridge IP                  | `false`  | N/A     |
+| `dns.servers`         | enables setting the DNS server to use                   | `false`  | N/A     |
+| `dns.searches`        | enables setting the DNS search domains to use           | `false`  | N/A     |
+| `experimental`        | enable experimental features                            | `false`  | N/A     |
+| `insecure_registries` | enables insecure registry communication                 | `false`  | N/A     |
+| `ipv6`                | enables IPv6 networking                                 | `false`  | N/A     |
+| `mtu`                 | enable setting the containers network MTU               | `false`  | N/A     |
```

Comment:
I noticed we use both `enable` and `enables` for the parameter descriptions.

Should we update it one way or the other to be consistent?

Reply:
Yeah, that's a good idea. I'll settle on `enable`